tests/testthat/test-include-diabetes-diagnoses.R

```diff
@@ -3,78 +3,82 @@ test_input_from_lpr2 <- tibble::tribble(
-  "00002", as.Date("2015-01-10"),              0,                    0,               0,               1,              TRUE,
-  "00002", as.Date("2021-01-10"),              0,                    0,               0,               1,              TRUE,
-  "00004", as.Date("2021-07-01"),              0,                    0,               0,               0,              TRUE,
-  "00004", as.Date("2021-08-01"),              0,                    0,               0,               0,              TRUE,
-  "00005", as.Date("2022-01-01"),              1,                    0,               0,               0,              TRUE,
-  "00006", as.Date("2015-02-01"),              1,                    1,               1,               1,              TRUE,
-  "00006", as.Date("2015-03-01"),              1,                    1,               1,               1,              TRUE
-)
-
-# Run tests
-
-actual_output <- include_diabetes_diagnoses(test_input_from_lpr2, test_input_from_lpr3)
-
-# Sort both tibbles for stable comparison
-actual_output_sorted    <- dplyr::arrange(actual_output,  pnr, dates)
-expected_output_sorted  <- dplyr::arrange(expected_output, pnr, dates)
+  ~pnr,    ~date,        ~n_t1d_endocrinology, ~n_t2d_endocrinology, ~n_t1d_medical, ~n_t2d_medical, ~has_lpr_diabetes_diagnosis,
+  "00001", "2015-01-01",              3,                    0,               0,               0,              TRUE,
+  "00001", "2015-02-01",              3,                    0,               0,               0,              TRUE,
+  "00002", "2015-01-10",              0,                    0,               0,               1,              TRUE,
+  "00002", "2021-01-10",              0,                    0,               0,               1,              TRUE,
+  "00004", "2021-07-01",              0,                    0,               0,               0,              TRUE,
+  "00004", "2021-08-01",              0,                    0,               0,               0,              TRUE,
+  "00005", "2022-01-01",              1,                    0,               0,               0,              TRUE,
+  "00006", "2015-02-01",              1,                    1,               1,               1,              TRUE,
+  "00006", "2015-03-01",              1,                    1,               1,               1,              TRUE
+) |>
+  dplyr::mutate(date = as.Date(date))
 
 # Test
-test_that("Filtering and counting diabetes diagnoses",
-          {
-            expect_equal(actual_output_sorted, expected_output_sorted)
-          })
+test_that("Filtering and counting diabetes diagnoses", {
+  actual_output <- include_diabetes_diagnoses(
+    test_input_from_lpr2,
+    test_input_from_lpr3
+  )
+
+  # Sorted stable comparison
+  actual_output_sorted <- dplyr::arrange(actual_output, pnr, date)
+  expected_output_sorted <- dplyr::arrange(expected_output, pnr, date)
```

Comment:
```suggestion
  actual <- include_diabetes_diagnoses(
    test_input_from_lpr2,
    test_input_from_lpr3
  )

  # Sorted stable comparison
  actual_sorted <- dplyr::arrange(actual, pnr, date)
  expected_sorted <- dplyr::arrange(expected, pnr, date)
```
Also to align with the naming in other test files :)